### ISSUE TITLE
Any possibility to publish new version?

### ISSUE THREAD

**Issue by Valian** [qgadrian/elixir_git_hooks#201]
Hi 👋 thanks for an amazing library!

I've run into some dependency problems with recase that seems to be updated on `master`:

```
Because smokestack >= 0.9.0 depends on recase ~> 0.8 and git_hooks >= 0.5.2 depends on recase ~> 0.7.0, smokestack >= 0.9.0 is incompatible with git_hooks >= 0.5.2.
And because your app depends on git_hooks ~> 0.7, smokestack >= 0.9.0 is forbidden.
So, because your app depends on smokestack ~> 0.9, version solving failed.
```

Basically recase dependency is a bit strict and forbids installation of other packages using newer versions of `recase`. On the other hand, master is correctly updated thanks to dependabot. 

It looks like lib was released last time 2 years ago. Would be great to get a new release with all the goodies & updated dependencies!

**Comment by qgadrian** [comment]
Hello @Valian 

I published a new version [`v0.7.4`](https://github.com/qgadrian/elixir_git_hooks/releases/tag/v0.7.4) that includes deps updated

**Comment by Valian** [comment]
Thank you a lot!
